topics/variant-analysis/tutorials/variant-calling/tutorial.md

```diff
@@ -0,0 +1,605 @@
+---
+layout: tutorial_hands_on
+title: "Variant Calling Workflow"
+questions:
+  - "How do I find sequence variants between my sample and a reference genome?"
+objectives:
+  - "Understand the steps involved in variant calling."
+  - "Describe the types of data formats encountered during variant calling."
+  - "Use command line tools to perform variant calling."
+time_estimation: "50 min"
+key_points:
+  - "Bioinformatic command line tools are collections of commands that can be used to carry out bioinformatic analyses."
+  - "To use most powerful bioinformatic tools, you will need to use the command line."
+  - "There are many different file formats for storing genomics data. It is important to understand what type of information is contained in each file, and how it was derived."
+contributors:
+  - Sofokli5
+  - fpsom
+  - shiltemann
+  - hexylena
+  - carpentries
+---
+
+
+# Introduction
+{:.no_toc}
+
+> ### {% icon comment %} Comment
+>
+> This tutorial is significantly based on [the Carpentries](https://carpentries.org) ["Data Wrangling and Processing for Genomics"](https://datacarpentry.org/wrangling-genomics/) lesson
+>
+{: .comment}
+
+In this tutorial we are working with files from a long-term evolution study of an *E. coli* population (designated Ara-3). We will perform variant calling to see how the population changed over time. We care how this population changed relative to the original population, *E. coli* strain REL606. Therefore, we will align each of our samples to the *E. coli* REL606 reference genome, and see what differences exist in our reads versus the genome.
+
+The whole proccess is designed to be run in `RStudio` ,as an `Active Interactive Tool`, directly in `Galaxy`.
+
+# {% icon hands_on %} Installing software

```

Comment:
this entire hands_on part should be prefixed with '>' and have an `{: .hands_on}` after it to render correctly (the tip boxes can be nested in it. So something like:


```
> ### {% icon hands_on %} Hands-on: Title of hands-on section
>
>  1. Step one
>  2. Step two
>
>  > ### {% icon tip %} Tip: title of tip
>  >  Tip: this is a great idea, try it!
>  {: .tip}
>
{: .hands_on}
```

Reply:
same for most of the other boxes btw

Reply:
Very usefull comment! I checked and corrected every box.